src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateArr = [];
+  let resultString = '';
+  let index = 0;
+
+  dateArr = date.split(fromFormat[3]);
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    const fromIndex = fromFormat.indexOf(toFormat[i]);
+
+    index++;
+
+    if (toFormat[i] === 'YY' && fromFormat.includes('YYYY')) {
+      const number = dateArr[index - 1];
+
+      resultString += number.slice(-2);
+    } else if (toFormat[i] === 'YYYY' && fromFormat.includes('YY')) {
+      const year = Number(dateArr[index - 1]);
+
+      if (dateArr[index - 1] === '00') {
+        resultString += '2000.';
+        continue;
+      } else {
+        const fullYear = year < 30 ? `20${year}` : `19${year}`;
+
+        resultString += fullYear;
+      }
```

Comment:
This condition checks if the year is equal to '00'. However, the task conditions do not specify any special handling for this case. According to the task, if `YY` is less than 30, we should prepend '20' to it, so '00' should be converted to '2000', not '2000.'.